dotnet/src/Connectors/Connectors.MistralAI/Client/MistralUsage.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel.Connectors.MistralAI.Client;
+
+/// <summary>
+/// Usage for chat completion.
+/// </summary>
+public class MistralUsage
```

Comment:
consider moving this as internal, as the response base is also.

Reply:
This must be public so callers can read this information

Reply:
This is public intentionally because it provides access to the Usage information